store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9642,4 +9642,12 @@ TODO: delete them permanently from here
 <attr id="5003" name="zimbraCustomAppBanner" type="string" max="256" cardinality="single" optionalIn="account,cos,domain" flags="accountInfo,accountCosDomainInherited,domainAdminModifiable" since="8.8.6">
   <desc>Logo app banner URL for web client</desc>
 </attr>
+
+<attr id="5004" name="zimbraSmtpStartTlsTrustedHosts" type="string" cardinality="single" optionalIn="globalConfig,server,domain" flags="serverInherited" since="8.8.6">
+  <globalConfigValue>*</globalConfigValue>
+  <desc>Configures trusted hosts sending mail from mailboxd to MTA with starttls.
+        If set to "*", all hosts are trusted. If set to a whitespace separated list of hosts, those hosts are trusted.
+        This parameter doesn't affect on smtp sessions to zimbraDataSourceSmtpHost.
```

Comment:
nit: reword, perhaps...
This attribute does not affect data source SMTP connections as those are controlled by zimbraDataSourceSmtpHost, zimbraDataSourceSmtpConnectionType, etc.